app/src/main/java/com/samco/trackandgraph/ui/DurationInputView.kt

```diff
@@ -49,7 +51,11 @@ class DurationInputView : FrameLayout {
             }
             return@setOnEditorActionListener false
         }
-        binding.hoursInput.requestFocus()
+        binding.hoursInput.focusAndShowKeyboard()
```

Comment:
Have to be very careful here this is a re-usable view so everywhere (any screen or layout) that you add this view to this code will be called. This caused a bug on the add feature fragment where returning to the screen would sometimes bring up a numerical keyboard because there's a duration input view on that screen. This happened even though the view was not visible. Very confusing behaviour. Again i've simply removed this line. It's the fragments job to decide when to show the keyboard, not the view its self.